templates/drone/4/rancher-compose.yml

```diff
@@ -0,0 +1,183 @@
+      type: enum
+      default: "release"
+      options:
+        - "release"
+        - "debug"
+      required: true
+    - variable: drone_open
+      label: Open Registration
+      description: |
+        Users self register. http://readme.drone.io/admin/user-registration/
+      required: true
+      default: true
+      type: enum
+      options:
+        - true
+        - false
+    - variable: drone_admin
+      label: Drone Admin
+      description: List of admins for drone comma seperated. http://readme.drone.io/admin/user-admins/
+      type: string
+      required: false
+    - variable: drone_orgs
+      label: Organizations
+      description: Comman seperated list of org that can access drone. http://readme.drone.io/admin/user-registration/
+      type: string
+      required: false
+    - variable: "drone_driver"
+      type: "enum"
+      required: true
+      label: "Remote Driver"
+      default: "GitHub"
+      description: "Remote Git and Auth scheme. ref http://readme.drone.io/admin"
+      options:
+        - GitHub
+        - GitLab
+        - Gogs
+        - Bitbucket Cloud
+        - Bitbucket Server
+    - variable: drone_driver_client
+      label: Remote Driver Client
+      description: "Client key from remote driver. Required for GitHub, Bitbucket Cloud, Bitbucket Server and GitLab."
+      type: string
+      required: false
+    - variable: drone_driver_secret
+      label: Remote Driver Secret
+      description: "Secret key from remote driver. Required for GitHub, Bitbucket Cloud, Bitbucket Server and GitLab."
+      type: multiline
+      required: false
+    - variable: drone_driver_url
+      label: Remote Driver URL
+      description: "Remote Driver server url. Required for GitLab, Gogs and Bitbucket Server; see http://readme.drone.io/admin."
+      type: string
+      required: false
+    - variable: drone_driver_user
+      label: Remote Driver Username
+      description: "Remote Driver username. Required for BitBucket Server; see http://docs.drone.io/install-for-bitbucket-server/."
+      type: string
+      required: false
+    - variable: drone_driver_password
+      label: Remote Driver Password
+      description: "Remote Driver password. Required for BitBucket Server, http://docs.drone.io/install-for-bitbucket-server/."
+      type: password
+      required: false
+    - variable: drone_server
+      label: Drone Server
+      description: "Drone sever identifier. Used by the agent to connect to the server (does not require change)."
+      type: string
+      default: "drone:9000"
+    - variable: drone_server_image
+      label: Drone Server Image
+      description: "Docker image used for the Drone server container."
+      type: string
+      default: "drone/drone:0.8.0-rc.1"
+    - variable: drone_agent_image
+      label: Drone Agent Image
+      description: "Docker image used for the Drone agent container(s)."
+      type: string
+      default: "drone/agent:0.8.0-rc.1"
```

Comment:
Image version should be fixed in the docker compose definition. Rancher compose already defines the drone version. 

If you still want to do in this way, please join server and client version in just one variable. Also, se version enum instead of string, fixed version prefered.

```
-    - variable: drone_server_image
-      label: Drone Server Image
-      description: "Docker image used for the Drone server container."
-      type: string
-      default: "drone/drone:0.8.0-rc.1"
-    - variable: drone_agent_image
-      label: Drone Agent Image
-      description: "Docker image used for the Drone agent container(s)."
-      type: string
-      default: "drone/agent:0.8.0-rc.1"
+    - variable: drone_version
+      label: Drone version
+      description: "Docker version used for the Drone container(s)."
+      type: enum
+      default: "0.8.0-rc.1"
+      options:
+        - 0.8.0-rc.1
```